packages/grafana-ui/src/components/Logs/LogRows.test.tsx

```diff
@@ -93,10 +93,88 @@ describe('LogRows', () => {
 
     expect(wrapper.find(LogRow).length).toBe(100);
   });
+
+  it('renders asc ordered rows if order and function supplied', () => {
+    const rows: LogRowModel[] = [
+      makeLog({ uid: '1', timeEpochMs: 1 }),
```

Comment:
I think it would make sense to change the order of this array to make sure that the ordering works well.

Reply:
Yeah! Great point. Updated in https://github.com/grafana/grafana/pull/26669/commits/8cc825a76add3576a87e6369064a2ee80d99b1d7